tests/containers/image.pm

```diff
@@ -81,7 +81,7 @@ sub run {
     # We may test either one specific image VERSION or comma-separated CONTAINER_IMAGE_VERSIONS
     my $versions = get_var('CONTAINER_IMAGE_VERSIONS', get_required_var('VERSION'));
     for my $version (split(/,/, $versions)) {
-        my $image = get_image_uri(version => $version);
+        my $image = get_image_uri(version => $version, distri => ($version =~ /-SP/ ? 'sle' : undef()));
 
```

Comment:
This update is needed when HDD DISTRI is different from containers distri inCONTAINER_IMAGE_VERSIONS, like in this SLE-M cases. the routine get by default test's `DISTRI`

Reply:
sle 16 does not have any `SP` that might be a future potential problem

Reply:
Ops, I was not aware of such changes. But, this applies to `CONTAINER_IMAGE_VERSIONS` items only, like `15-SP2,15-SP3,15-SP4,...`: how should we expect that list with 16?

Reply:
But eventually in that case, we'll add a more condition : `($version =~ /-SP|16/ ? ...`

Reply:
From what I see this is the only code change that is actually fixing something, I would make this alone a PR and split the other changes also into different PRs.

Reply:
How is this fixing something? Also, keep in mind that we also need to run this on openSUSE, so this might break stuff over there.

Reply:
No, this is not the only code change.
Here is needed too: 
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20383/files#diff-8a38b5d8cac8d3a4ff4881bd952d410f3d640aacd3dedceea2fedea2095f46f9R185

Reply:
and yes: it is TL;DR to explain, but update is tricky and needed, due to the BCI Container test flows